src/main/java/seedu/address/model/tag/Tag.java

```diff
@@ -9,7 +9,7 @@
  */
 public class Tag {
 
-    public static final String MESSAGE_CONSTRAINTS = "Tags names should be alphanumeric";
+    public static final String MESSAGE_CONSTRAINTS = "Tags cannot be an empty string!";
```

Comment:
Would `Tags cannot be empty!` be more user-friendly than `Tags cannot be an empty string!`? I am okay with both phrasings.

Reply:
I have thought of the former, but im not too sure whether there will be confusion since our tag field is optional

Reply:
Okay sure!